src/reducers/transactionReducer.ts

```diff
@@ -90,6 +97,15 @@ export default function transactionReducer(state: TransactionState, action: Tran
     }
     case TransactionActionTypes.SET_CUSTOM_CALL_INPUT: {
       const { customCallInput, createType, targetChain } = action.payload;
+      if (!customCallInput) {
+        return {
+          ...state,
+          customCallInput,
+          transactionReadyToExecute: false,
+          customCallError: null,
+          shouldEvaluatePayloadEstimatedFee: false
+        };
+      }
```

Comment:
How is this related?

Reply:
When inputs are cleared ( because of this process or manual ) there is an error message which in my opinion is unnecessary.  The expected behavior is that when input gets cleared for whatever reason the execution button should not be available, the fee should not be calculated and no error message should appear on the input.

Reply:
I think it is related in the way that is part of the reset process which is the whole point of this PR, even if it's not 100% related to debounce text field component.